package.json

```diff
@@ -88,6 +88,7 @@
     "./package.json": "./package.json",
     "./styles.css": "./src/styles.css",
     "./tailwind.css": "./src/tailwind.css",
+    "./theme": "./esm/styles/theme.js",
```

Comment:
this is gonna be super cool where we expose these themes for users to do basic style editing.